src/WebJobs.Script.WebHost/WebHostServiceCollectionExtensions.cs

```diff
@@ -214,7 +218,7 @@ private static void AddLinuxContainerServices(this IServiceCollection services)
             services.AddSingleton<IHostedService>(s =>
             {
                 var environment = s.GetService<IEnvironment>();
-                if (environment.IsLinuxConsumption())
+                if (environment.IsLinuxConsumptionOnAtlas() || environment.IsLinuxConsumptionOnLegion())
```

Comment:
IsAnyLinuxConsumption()?

Reply:
same as https://github.com/Azure/azure-functions-host/pull/8617#discussion_r942991081
basically any code using Any will stay that way
any code using specific checks will need to change.

Reply:
Then please put TODOs, since as it stands if this PR is checked in people will have the same question I did :)